### PR TITLE
add Lambda 2M support

### LINKED ISSUE #150: ophyd support for new Lambda 2M detector
We'll need to create a device for the new 2M detector.  Use standard `ophyd.areaDetector` components as much as possible.

### PR DESCRIPTION
**Pull request by prjemian**
- FIX #150

### CODE REVIEW

**Comment by prjemian** [comment]
@qzhang234 This is ready for testing **before** merging.

File: profile_bluesky/startup/instrument/devices/ad_eiger.py [hunk 1]
```diff
@@ -99,7 +100,47 @@ def _acquire_changed(self, value=None, old_value=None, **kwargs):
...
+        # Ensure we do not have an old file open.
+        set_and_wait(self.capture, 0)
+        # These must be set before parent is staged (specifically
+        # before capture mode is turned on. They will not be reset
+        # on 'unstage' anyway.
+        set_and_wait(self.file_path, write_path)
+        set_and_wait(self.file_name, filename)
+        # set_and_wait(self.file_number, 0)
+
+        # get file number now since it is incremented during stage()
```
Comment (prjemian) [inline profile_bluesky/startup/instrument/devices/ad_eiger.py:121]: You could probably be OK to comment out these two lines.

**Comment by prjemian** [comment]
I'm ok with merge since we'll keep the branch open.

**Comment by prjemian** [comment]
@qzhang234 It wqould be good to "tag" the repository now, since the Eiger is working.

**Comment by qzhang234** [comment]
@prjemian  I'm not sure what 'tagging the repository' means. Could you elaborate?

**Comment by prjemian** [comment]
I did it. look at teams.

**Pull request merged**